scripts/download_superglue_data.py

```diff
@@ -47,6 +48,13 @@ def download_diagnostic(data_dir):
         os.mkdir(os.path.join(data_dir, "RTE"))
```

Comment:
The print above will print twice now with the same message, even if you make each fn call only download one dataset.